k8s-artifacts/mg-enforcer/service.yaml

```diff
@@ -0,0 +1,33 @@
+# --------------------------------------------------------------------
+# Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------
+
+# Service for enforcer
+
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: mg-enforcer
+  name: enforcer-service
+spec:
+  type: ClusterIP
+  selector:
+    app: mg-enforcer
+  ports:
+    - name: "8081"
```

Comment:
Lets use proper names

Reply:
Fixed in 6b6708a8472501213c00230342810f801d5df4cf